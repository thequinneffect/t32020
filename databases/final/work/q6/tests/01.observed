tests/01.sh: 1: ./q6: Permission denied
