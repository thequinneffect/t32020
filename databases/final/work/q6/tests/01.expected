Discography for Poor Knights
--------------------
Shock Fear (2012) (blues)
--------------------
 1. Dreams of Haze (5:19)
 2. Ode to the Dance Floor (11:18)
 3. Waiting for the Horizon (3:33)
 4. Bad for the Universe (8:02)
 5. Run with Summer (1:19)
 6. Where is the Torpedoes (6:37)
 7. Bad for the Moon (4:26)
 8. Beyond this Night (3:20)
 9. Why not the Farm (9:35)
10. Shock the Sky (4:24)
